src/test/resources/org/prebid/server/it/openrtb2/yeahmobi/test-auction-yeahmobi-response.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "tid",
+  "seatbid": [
+    {
+      "bid": [
+        {
+          "adm": "<b>hi</b>",
+          "cid": "test_cid",
+          "crid": "test_banner_crid",
+          "ext": {
+            "bidder": {
+              "format": "BANNER"
+            },
+            "prebid": {
+              "type": "banner",
+              "targeting": {
+                "hb_bidder": "yeahmobi",
+                "hb_bidder_yeahmobi": "yeahmobi",
+                "hb_cache_host": "{{ cache.host }}",
+                "hb_cache_host_yeahmobi": "{{ cache.host }}",
+                "hb_cache_id": "f0ab9105-cb21-4e59-b433-70f5ad6671cb",
+                "hb_cache_id_yeahmobi": "f0ab9105-cb21-4e59-b433-70f5ad6671cb",
+                "hb_cache_path": "{{ cache.path }}",
+                "hb_cache_path_yeahmobi": "{{ cache.path }}",
+                "hb_pb": "0.00",
+                "hb_pb_yeahmobi": "0.00"
+              },
+              "cache": {
+                "bids": {
+                  "cacheId": "f0ab9105-cb21-4e59-b433-70f5ad6671cb",
+                  "url": "{{ cache.resource_url }}f0ab9105-cb21-4e59-b433-70f5ad6671cb"
+                }
+              }
+            }
+          },
+          "id": "1",
+          "impid": "imp123",
```

Comment:
In request you have impId = impId001, in response imp123. The same for cache and bid response json files

Reply:
Yeah, thank you for notice and attention!
This is the mistake in test-yeahmobi-bid-response.json. That caused bad results in other test json's.
I fixed it.